CMakeLists.txt

```diff
@@ -372,10 +366,24 @@ if(UMF_FORMAT_CODE_STYLE)
         COMMENT "Format files using clang-format and cmake-format")
 endif()
 
-# Add license to the installation path
+# --------------------------------------------------------------------------- #
+# Configure make install/uninstall and packages
+# --------------------------------------------------------------------------- #
 install(FILES ${CMAKE_SOURCE_DIR}/LICENSE.TXT
         DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/doc/${PROJECT_NAME}/")
 
+install(
+    DIRECTORY examples/
+    DESTINATION ${CMAKE_INSTALL_DOCDIR}/examples
+    FILES_MATCHING
+    PATTERN "*.c")
```

Comment:
one issue to solve: should we install only `c` files or should we also include directories, for each example.

On `main` we install `basic/basic.c`
On my PR we install `basic.c`

Which is expected? At the moment, this is the reason to fail the CI...

Reply:
The current layout with subdirs will be most useful if there will be  examples with multiple files. However, if there is slight chance that we'll create examples with more than one file, then putting all source files into one directory would be a better solution.
I think we could put all examples in one directory now and eventually change this in the future, if needed.

Reply:
@lukaszstolarczuk I agree with @PatKamin

Reply:
I couldn't quickly make it work with the `PATTERN` approach. I think we would have to use `glob` to iterate over all examples' dirs or something.

That's why, since we only have one example now, I made it simple like it was before - just copy a file to the proper place.

Please let me know, what you think.